repositories/memory/topics/get_topic_test.go

```diff
@@ -0,0 +1,98 @@
+package topics
+
+import (
+	"context"
+	"github.com/stretchr/testify/assert"
+	"testing"
+
+	"github.com/matheusmosca/walrus/domain/entities"
+	"github.com/matheusmosca/walrus/domain/vos"
+	"github.com/stretchr/testify/require"
+)
+
+type args struct {
+	topicName vos.TopicName
+}
+type testCase struct {
+	description string
+	args        args
+	beforeRun   func(storage map[vos.TopicName]entities.Topic)
+	wantErr     error
```

Comment:
you're should also check if the topic that the function returns if the expected, so you can create a "want" with type vos.TopicName

Reply:
I tried this but it doesn't allow me to do the same because I am unable to get `newTopic.name` for some reason. Guide me with this or we can let this be since it doesn't affect the UT.

Reply:
Sure, your branch in not updated with the main branch. I'll do this improvement later

Reply:
Should I re base it? I dont see any conflicts